--- qt-everywhere-opensource-src-5.1.0/qtwebkit/Source/WebCore/platform/graphics/gstreamer/PlatformVideoWindowPrivate.h.orig    2013-07-25 23:05:48.082002578 +0200
+++ qt-everywhere-opensource-src-5.1.0/qtwebkit/Source/WebCore/platform/graphics/gstreamer/PlatformVideoWindowPrivate.h 2013-07-25 23:06:32.038747918 +0200
@@ -25,6 +25,8 @@

 #include <QWindow>

+#if ENABLE(VIDEO) && USE(GSTREAMER) && !defined(GST_API_VERSION_1)
+
 namespace WebCore {

 class HTMLVideoElement;
@@ -55,5 +57,7 @@

 } // namespace WebCore

+#endif /* #if ENABLE(VIDEO) && USE(GSTREAMER) && !defined(GST_API_VERSION_1) */
+

 #endif // PlatformVideoWindowPrivate_h
