diff -Nurp a/configure.ac b/configure.ac
--- a/configure.ac	2013-04-15 18:19:58.014124098 +0200
+++ b/configure.ac	2013-04-15 18:19:37.474380498 +0200
@@ -1233,6 +1233,7 @@ AM_CONDITIONAL([ENABLE_CSS_SHADERS],[tes
 
 # Gtk conditionals
 AM_CONDITIONAL([ENABLE_INTROSPECTION],[test "$enable_introspection" = "yes"])
+AM_CONDITIONAL([ENABLE_GTK_DOC],[test "$enable_gtk_doc" = "yes"])
 
 AC_CONFIG_FILES([
 GNUmakefile
