diff -Nurp a/qtdeclarative-opensource-src-5.1.1/examples/qmltest/qmltest/qmltest.pro b/qtdeclarative-opensource-src-5.1.1/examples/qmltest/qmltest/qmltest.pro
--- a/qtdeclarative-opensource-src-5.1.1/examples/qmltest/qmltest/qmltest.pro	2014-02-11 11:19:59.194804855 +0100
+++ b/qtdeclarative-opensource-src-5.1.1/examples/qmltest/qmltest/qmltest.pro	2014-02-11 11:20:30.486410171 +0100
@@ -14,7 +14,7 @@ TESTDATA += tst_basic.qml tst_item.qml
 # This code exists solely for the purpose of building this example
 # inside the examples/ hierarchy.
 
-QT += qml qmltest
+QT += qml qmltest quick
 
 macx: CONFIG -= app_bundle
 
