diff -Nurp a/qtdeclarative-opensource-src-5.1.0-beta1/tools/qmltestrunner/qmltestrunner.pro b/qtdeclarative-opensource-src-5.1.0-beta1/tools/qmltestrunner/qmltestrunner.pro
--- a/qtdeclarative-opensource-src-5.1.0-beta1/tools/qmltestrunner/qmltestrunner.pro	2013-05-13 05:41:26.000000000 +0200
+++ b/qtdeclarative-opensource-src-5.1.0-beta1/tools/qmltestrunner/qmltestrunner.pro	2013-05-29 19:17:23.790934142 +0200
@@ -1,5 +1,5 @@
 SOURCES += main.cpp
 
-QT += qml qmltest
+QT += qml qmltest quick
 
 load(qt_tool)
